.gitmodules

```diff
@@ -37,7 +37,7 @@
 	url = https://github.com/axboe/liburing.git
 [submodule "third_party/mlir-hlo"]
 	path = third_party/mlir-hlo
-	url = https://github.com/tensorflow/mlir-hlo.git
+	url = https://github.com/google/iree-llvm-fork.git
```

Comment:
This seems like the wrong repo to be updating?

Reply:
This is the right repo. It is where we store our patched branches for LLVM derived projects.